exodus/trackers/management/commands/import_from_etip.py

```diff
@@ -0,0 +1,112 @@
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from trackers.models import Tracker
+
+ETIP_API_DEFAULT_HOSTNAME = 'https://etip.exodus-privacy.eu.org'
+ETIP_API_PATH = '/api/trackers'
```

Comment:
Can this variables be set in `exodus/exodus/settings/base.py` instead ?
I think this will be more convenient for future uses and Docker environment variables :)

Reply:
Sure, good idea, I'll change that!